apps/judicial-system/backend/src/app/formatters/custodyNoticePdf.ts

```diff
@@ -51,22 +57,22 @@ export async function getCustodyNoticePdfAsString(
     })
     .text(' ')
     .font('Helvetica-Bold')
-    .fontSize(14)
+    .fontSize(mediumFontSize)
     .lineGap(8)
     .text('Sakborningur')
-    .fontSize(12)
+    .fontSize(baseFontSize)
```

Comment:
This will change from font size 12 to font size 11.
Is that what we want here?

Reply:
Same ☝️